test/e2e/volumes.go

```diff
@@ -393,6 +394,72 @@ var _ = framework.KubeDescribe("Volumes [Feature:Volumes]", func() {
 			// Must match content of test/images/volumes-tester/nfs/index.html
 			testVolumeClient(cs, config, volume, nil, "Hello from NFS!")
 		})
+
+		It("should be mountable after checking mount binaries", func() {
```

Comment:
Also how is this test different from the standard NFS test above and how did you verify that it is working?